### PR TITLE
293_post_interest_job

### PR DESCRIPTION
**Pull request by AvinashKumarG**

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsSchedularServiceImpl.java [hunk 1]
```diff
@@ -37,42 +40,66 @@
...
+	public void postInterestForAccounts() throws JobExecutionException {
+		final int maxPazeSize = 100;
+		boolean hasErrorOccurred = false;
+		int offSet = 1;
+		Integer initialSize = 100;
+		final Integer status = SavingsAccountStatusType.ACTIVE.getValue();
+		
+		
+		do {
+			Page<SavingsAccountData> savingsAccounts = this.savingAccountReadPlatformService.retrieveSavingsAccount(maxPazeSize, offSet,
```
Comment (nazeer1100126) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsSchedularServiceImpl.java:66]: By reading Savings Account Entities page wise, you can avoid lot of db calls.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsSchedularServiceImpl.java [hunk 2]
```diff
@@ -37,42 +40,66 @@
...
-        
-        if (sb.length() > 0) { throw new JobExecutionException(sb.toString()); }
-    }
+	@Transactional
+	@CronTarget(jobName = JobName.POST_INTEREST_FOR_SAVINGS)
+	@Override
+	public void postInterestForAccounts() throws JobExecutionException {
+		final int maxPazeSize = 100;
+		boolean hasErrorOccurred = false;
+		int offSet = 1;
```
Comment (avikganguly01) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsSchedularServiceImpl.java:60]: Offset should be 0

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsSchedularServiceImpl.java (hunk 1 above)
Comment (avikganguly01) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsSchedularServiceImpl.java:66]: @nazeer1100126 : The reads are page-wise only. I did not understand your concern.

**Comment by avikganguly01** [comment]
@AvinashKumarG : I know it's a bit late but would it be possible for you to resolve the conflict, test it once and update your PR?

**Comment by myrlen** [comment]
PR ping.  Is this Pull Request still alive?

**Comment by myrle-krantz** [comment]
ping

Is this pull request still "alive"? Will Nazeer and Avik's question receive an answer? If there's no response in a week this pull request will be closed.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsSchedularServiceImpl.java (hunk 1 above)
Comment (nazeer1100126) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsSchedularServiceImpl.java:66]: Check the complete flow. He is calling postInterestInBatch with Pojos and then he is again reading the entities from DB. My point was to use JPA pagable implementation

**Comment by vorburger** [comment]
@AvinashKumarG thank you for your interest in Mifos.  I'm an (old) volunteer who got reminded of Mifos this week-end at the FOSDEM conference, and I'm taking a moment to help the project clean up some of it's old issues and pull requests etc.  

Given that there was no reaction to above in over 1 year, I took the liberty to close this PR, to avoid confusion for future new contributors looking at this repository - hope you understand and that's fair.

**Pull request closed without merging**